src/XMakeBuildEngine/BackEnd/Components/Communications/NodeProviderInProc.cs

```diff
@@ -203,19 +203,19 @@ public bool CreateNode(int nodeId, INodePacketFactory factory, NodeConfiguration
             // Attempt to get the operating environment semaphore if requested.
             if (_componentHost.BuildParameters.SaveOperatingEnvironment)
             {
-                // We can only create additional in-proc nodes if we have decided not to save the operating environment.  This is the global
-                // DTAR case in Visual Studio, but other clients might enable this as well under certain special circumstances.
```

Comment:
Can you preserve this comment? It sounds like it's related to the codepath, not the line below that you're removing.